engine/src/main/java/org/terasology/engine/entitySystem/sectors/SectorUtil.java

```diff
@@ -46,7 +46,7 @@ public static SectorRegionComponent createSectorRegionComponent(Collection<Vecto
      */
     public static void addChunksToRegionComponent(SectorRegionComponent regionComponent, Collection<Chunk> chunks) {
         regionComponent.chunks.addAll(chunks.stream()
-                .map(k -> k.getPosition(new Vector3i()))
+                .map(k -> new Vector3i(k.getPosition()))
```

Comment:
Do we need this `new`?

Reply:
its in SectorRegionComponent so you don't know if those might change.

Reply:
hmm, I _think_ it would be okay if regionComponent.chunks changed to Vector3i**c**, but I'm okay with saying that doesn't have to be in the scope of this PR. Seems like there's other doubts about SectorRegion's future.